content/hi/docs/setup/best-practices/node-conformance.md

```diff
@@ -35,47 +35,49 @@
 
 2. आदेश के साथ नोड अनुरूपता परीक्षण चलाएँ:
 
-`` `खोल
+```shell
 # $CONFIG_DIR आपके क्यूबलेट का पॉड मेनिफेस्ट पथ है।
 # $LOG_DIR परीक्षण आउटपुट पथ है।
-sudo docker रन -it --rm --विशेषाधिकार प्राप्त --net=host \
-   -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
-   k8s.gcr.io/node-test:0.2
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  k8s.gcr.io/node-test:0.2
 ```
 
 ## अन्य आर्किटेक्चर के लिए नोड अनुरूपता परीक्षण चलाना
-```
+
 कुबेरनेट्स अन्य के लिए नोड अनुरूपता परीक्षण डॉकर चित्र भी प्रदान करता है
 वास्तुकला:
 
- मेहराब  |       छवि     |
+  Arch  |       Image       |
 --------|:-----------------:|
  amd64  |  node-test-amd64  |
   arm   |    node-test-arm  |
  arm64  |  node-test-arm64  |
 
-   ## चयनित परीक्षण चल रहा है
+## चयनित परीक्षण चल रहा है
 
- विशिष्ट परीक्षण चलाने के लिए, पर्यावरण चर `FOCUS` को के साथ अधिलेखित करें
+विशिष्ट परीक्षण चलाने के लिए, पर्यावरण चर `FOCUS` को के साथ अधिलेखित करें
 उन परीक्षणों की नियमित अभिव्यक्ति जिन्हें आप चलाना चाहते हैं।
-`` `खोल
-sudo docker रन -it --rm --विशेषाधिकार प्राप्त --net=host \
-   -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
-   -ई फोकस = मिररपॉड \ # केवल मिररपॉड टेस्ट चलाएं
-   k8s.gcr.io/node-test:0.2
+
+
+```shell
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e FOCUS=MirrorPod \ # केवल मिररपॉड टेस्ट चलाएं
```

Comment:
@ShivamTyagi12345 Can you check the english docs again, you've removed an english word in the comments.

Reply:
Sure Thing @mittalyashu .Thanks for reviewing